docs/examples/expressions/expr_conditional_and_masking.ipynb

```diff
@@ -0,0 +1,386 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Conditional-like execution and masking\n",
+    "\n",
+    "This notebook shows how DALI arithmetic expressions can be utilized to achieve conditional-like application of augmentations and be used for some of the masking operations."
```

Comment:
```suggestion
    "This notebook shows how DALI arithmetic expressions can be used to achieve conditional-like application of augmentations and be used for some of the masking operations."
```
since you are using "utilized" below

Reply:
done